doc/generated/pmem_ctl.5

```diff
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-ctl \[en] interface for examination and modification of the library's
```

Comment:
You should not change those files by hand, only bot can generate new versions of manpages in this directory.

Reply:
Yeah, there's a check against that on Travis.  Which is bad as the bot doesn't handle 1.5 yet.